server/controller/Songs/songHandler.js

```diff
@@ -1,26 +1,57 @@
-const sqlQuery = require('../../database/my_sql_query');
-const dbConnection = require('../../database/db_connection');
+const sqlQuery = require("../../database/my_sql_query");
+const dbConnection = require("../../database/db_connection");
 
-// Get all songs    
-module.exports.getAllSongs = async (req, res) => {
-    try {
-        let connection = await dbConnection();
-        let getSongsQuery = "SELECT songname, uploadDate, numberOfLikes, duration FROM `songs`"
-        let getSongs = await sqlQuery(connection, getSongsQuery);
+import { SONG_ATTRIBUTE } from "./attribute";
+import { getRandomInt } from "../../helper/id";
 
-        connection.end();
-        return res.status(200).send(getSongs);
-    } catch (error) {
-        console.log(error);
-        res.status(500).json({
-            message: 'error',
-            error: error
-        })
-    }
+// Get all songs
+module.exports.getAllSongs = async (data, res) => {
+  try {
+    let connection = await dbConnection();
+    let getSongsQuery =
+      "SELECT songname, uploadDate, numberOfLikes, duration FROM `songs`";
```

Comment:
I have already adjust naming convention in MySQL. Please check it and test again! @Dev01-vinamr-Ken